src/executables/master_server_monitor.cpp

```diff
@@ -118,20 +111,18 @@ server_monitor::register_server_entry (int pid, const std::string &exec_path, co
       entry->second.set_pid (pid);
       entry->second.set_need_revive (false);
       entry->second.set_registered_time (std::chrono::steady_clock::now ());
-      fprintf (stdout,
-	       "[SERVER_REVIVE_DEBUG] : server entry has been updated in master_Server_monitor : pid : %d, exec_path : %s, args : %s\n",
-	       pid,
-	       exec_path.c_str(), args.c_str());
+      _er_log_debug (ARG_FILE_LINE,
+		     "[Server Monitor] [%s] Server entry has been updated. (pid : %d)",
```

Comment:
```suggestion
		     "[Server Monitor] [%s] Server entry has been registered. (pid : %d)",
```

register, unregister, revive, confirm 단계가 표현되었으면 좋겠습니다. 
내부적으로는 해당 엔트리가 갱신되는 것이지만, 단계로 보면 register 단계이기 때문에 해당 문구가 들어가면 좋을 것 같네요.